R/advanced_search.R

```diff
@@ -20,7 +20,14 @@
 #' # All orphaned packages
 #' advanced_search(Maintainer = "ORPHANED")
 #'
-#' # Packages that match a certain field
+#' # Packages with both Hester and Wickham as authors
+#' advanced_search(Author = "Hester", Author = "Wickham")
+#' advanced_search("Author: Hester AND Author: Wickham")
+#' 
+#' # Packages with Hester as an Author, and Wickham in any field
+#' advanced_search(Author = "Hester AND Wickham")
```

Comment:
I think we could just leave this one out? It is confusing imo. Maybe write it as

```r
advanced_search(Author = "Hester", "Wickham")
```
and the one below as 

```r
advanced_search(Author = "Hester", "NOT Wickham")
```

and also add this example:

```r
advanced_search(Author = "Hester", Author = "NOT Wickham")
```
?